.gitignore

```diff
@@ -4,4 +4,8 @@ umd/
 coverage/
 .nyc_output/
 .vscode/
+.rts2_cache_cjs/
```

Comment:
Maybe we should do `.rts2*` and replace all these?

Reply:
Done